pyof/v0x04/common/utils.py

```diff
@@ -115,3 +115,14 @@ def new_message_from_header(header):
     message.header.length = header.length
 
     return message
+
+
+def unpack_message(buffer):
+    """Unpack the whole buffer, including header pack."""
+    hdr_size = Header().get_size()
+    hdr_buff, msg_buff = buffer[:hdr_size], buffer[hdr_size:]
+    header = Header()
+    header.unpack(hdr_buff)
+    message = new_message_from_header(header)
```

Comment:
Actually, the v0x04 file defines `new_message_from_header`. @macartur and I have discussed about the methods being very similar to each other and the possibility to make a common file to all versions in the future, but right now this was the only missing method in this file in particular.